test/flycheck-test.el

```diff
@@ -3197,8 +3197,8 @@ See https://github.com/flycheck/flycheck/issues/531 and Emacs bug #19206"))
   (flycheck-ert-with-env
       `(("GOPATH" . ,(flycheck-ert-resource-filename "language/go")))
     (flycheck-ert-should-syntax-check
-     "language/go/src/gosimple/gosimple.go" 'go-mode
-     '(5 6 warning "should omit values from range; this loop is equivalent to `for range ...` (S1005)"
+     "language/go/src/gosimple/gosimple1.go" 'go-mode
+     '(4 6 warning "should omit values from range; this loop is equivalent to `for range ...` (S1005)"
```

Comment:
I see that, but the `for` loop is on line 5 because you left a comment on line 4 in place of the `xs` declaration.  I ran your branch on my machine, and the test is failing.  Does it work for you?  (Integration tests are *not* tested on Travis, sadly)